test/integration/targets/vyos_interfaces/tests/cli/rtt.yaml

```diff
@@ -0,0 +1,76 @@
+---
+- debug:
+    msg: "START vyos_interfaces round trip integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- name: Apply the provided configuration (base config)
+  vyos_interfaces:
+    config:
+      - name: eth0
```

Comment:
same comment

Reply:
Even though eth0 is mentioned, these tests actually do not modify any attributes of eth0. Given the way overridden is designed to work, if we don't specify eth0 in the task, it'll actually end up modifying its attributes. If you look at the commands generated by this task, you will not find any command that concerns eth0.